src/main/java/core/mvc/JsonView.java

```diff
@@ -1,12 +1,38 @@
 package core.mvc;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.MediaType;
+
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.Map;
 
 public class JsonView implements View {
+    private static final Logger logger = LoggerFactory.getLogger(JsonView.class);
+
     @Override
     public void render(Map<String, ?> model, HttpServletRequest request, HttpServletResponse response) throws Exception {
+        response.setContentType(MediaType.APPLICATION_JSON_VALUE);
+        response.setCharacterEncoding("UTF-8");
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        String json = StringUtils.EMPTY;
 
+        if (model.size() == 1) {
+            String key = model.keySet().iterator().next();
+            Object value = model.get(key);
+
+            json = objectMapper.writeValueAsString(value);
```

Comment:
size 가 1 이라는 것을 이미 알고 있으면 키로 조회하지 않더라도 바로 조회할 수 있을 것 같아요 :)
```suggestion
        if (model.size() == 1) {
            Object value = model.values().iterator().next();

            json = objectMapper.writeValueAsString(value);
```